src/storage/tests/user-data.test.ts

```diff
@@ -1,7 +1,12 @@
+/**
+ * @jest-environment node
+ */
 // To run tests, execute `npm run test -- src/storage/user-data.test.ts` from the root directory
 import fs from 'fs';
 import { loadUserData } from '../user-data';
 
+jest.mock( 'fs' );
```

Comment:
> If we want to mock Node's built-in modules (e.g.: `fs` or `path`), then explicitly calling e.g. `jest.mock('path')` is **required**, because built-in modules are not mocked by default.
>
> – [Jest Manual Mocks documentation](https://jestjs.io/docs/manual-mocks#mocking-node-modules)